sdk/core/core-http/test/policies/bearerTokenAuthenticationPolicyTests.ts

```diff
@@ -73,6 +74,19 @@ describe("BearerTokenAuthenticationPolicy", function() {
     }
   });
 
+  it("tests that AccessTokenRefresher is working", async function() {
+    this.timeout(35000);
```

Comment:
no, please don't do this. There is no reason to make a test take 30+ seconds just because of a default timer. We should mock the clock with sinon if you want to test that, but not sit there and waste CPU cycles

Reply:
alright, I'll follow your advice here

Reply:
I believe I did it here: https://github.com/Azure/azure-sdk-for-js/pull/10085/commits/7f3192ea69401bce8d2689e0ec4bdb66b5b991c1 thank you again for the feedback!